### PR TITLE
Add dynatrace-oneagent-operator v0.7.0

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

**Comment by DTMad** [comment]
Hey @guangbochen !
This is the new PR from master

File: charts/dynatrace-oneagent-operator/v0.7.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: dynatrace-oneagent-operator
+description: The Dynatrace OneAgent Operator Helm chart for Kubernetes and Openshift
+icon: https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_512x512px.png
```
Comment (guangbochen) [inline charts/dynatrace-oneagent-operator/v0.7.0/Chart.yaml]: can u please help to upload the icon to the parent directly and reference it as `file://../Signet_Logo_RGB_CP_512x512px.png`, therefore air-gapped env user can view it properly, thanks

File: charts/dynatrace-oneagent-operator/v0.7.0/questions.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
+categories:
+- APM
+- Monitoring
+rancher_min_version: v2.4.0
```
Comment (guangbochen) [inline charts/dynatrace-oneagent-operator/v0.7.0/questions.yml:4]: can u please help to add
```
labels:
  io.rancher.certified: partner
```

File: charts/dynatrace-oneagent-operator/v0.7.0/Chart.yaml (hunk 1 above)
Comment (DTMad) [reply charts/dynatrace-oneagent-operator/v0.7.0/Chart.yaml]: Sure - done!

File: charts/dynatrace-oneagent-operator/v0.7.0/questions.yml (hunk 2 above)
Comment (DTMad) [reply charts/dynatrace-oneagent-operator/v0.7.0/questions.yml:4]: Done

**Comment by guangbochen** [comment]
LGTM, thanks.

**Comment by DTMad** [comment]
Hey @guangbochen !
Nice - thanks for your help.

If you approve I can merge it then if everything's fine

**Pull request merged**